buildSrc/src/main/java/dependencies/Dependencies.kt

```diff
@@ -97,7 +97,7 @@ object Dependencies {
     const val hamcrest = "org.hamcrest:hamcrest:2.2"
     const val powermock_module_junit4 = "org.powermock:powermock-module-junit4:2.0.9"
     const val powermock_api_mockito2 = "org.powermock:powermock-api-mockito2:2.0.9"
-    const val robolectric = "org.robolectric:robolectric:4.6.1"
```

Comment:
I had to downgrade robolectric for now because of this issue https://github.com/robolectric/robolectric/issues/6593 there is a workaround there but I don't like it so I prefer waiting for a better version of this dependency.